drivers/mtd/Kconfig

```diff
@@ -23,6 +23,14 @@ config HAVE_MTD_NATIVE
     help
         Indicates that a native MTD is present.
 
+config HAVE_MTD_SDCARD_DEFAULT
+    bool
+    imply MODULE_MTD_SDCARD if MODULE_MTD
+    imply MODULE_MTD_SDCARD_DEFAULT if MODULE_MTD
+    imply MODULE_SDCARD_SPI if MODULE_MTD
```

Comment:
@MrKevinWeiss, @leandrolanzieri I guess there is no way around this copy & paste of dependencies?